reconplogger.py

```diff
@@ -35,15 +35,15 @@
         }
     },
     'loggers': {
-        'plogger_plain': {
+        'reconplogger_plain': {
             'level': 'DEBUG',
             'handlers': ['plain'],
         },
-        'plogger_json': {
+        'reconplogger_json': {
             'level': 'DEBUG',
             'handlers': ['json'],
         },
-        'plogger': {
+        'reconplogger': {
             'level': 'DEBUG',
             'handlers': ['json'],
         },
```

Comment:
@nischalhp I think we should also remove the `reconplogger` item in `loggers`. It is identical to `reconplogger_json` which was added to be more explicit about its format. The other one was kept just for backward compatibility, but because of the rename, this is no longer the case.

Reply:
We could also consider renaming things a bit to make things clearer, e.g. formatters be: `plain_formatter`, `json_formatter`; handlers be: `plain_handler`, `json_handler`, `null_handler`; and loggers be: `plain_logger`, `json_logger`, `null_logger`. What do you think?

Reply:
@mauvilsa yes I think it makes complete sense. It is the right time to shift to this thought process. I will do the needful changes on the same PR.